sensor_msgs/CompressedImage image
sensor_msgs/CameraInfo camerainfo
---
Score score
